lib/web_ui/lib/src/engine/context_menu.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
The path of this file needs to be added here:
https://github.com/flutter/engine/blob/eb5c6f0b4ed94325e9c9811cc8681d687e0f5c8a/ci/licenses_golden/licenses_flutter#L1910-L1914